docs/content/docs/concepts/summarization.md

```diff
@@ -0,0 +1,306 @@
+---
+title: Summarization
+menuPosition: 10
+---
```

Comment:
Since you've changed the URL, you need to add an alias or the old URL will be broken. You can also just not rename the file and use the existing URL. There doesn't seem to be any reason to change it. You should also keep the existing aliases for the same reason.

```suggestion
---
title: Summarization
menuPosition: 10
aliases:
  - "/docs/advanced/summarizer/"
  - "/docs/deep/summarizer/"
  - "/docs/concepts/summarizer/"
---
```

Reply:
I didn't realize we need to keep aliases to old locations. Is it because if someone has a link to the old doc, it won't work?

Reply:
I changed the name back and added aliases.

Reply:
> I didn't realize we need to keep aliases to old locations. Is it because if someone has a link to the old doc, it won't work?

Yes. As Tim Berners-Lee himself said, "Cool URIs don't change": https://www.w3.org/Provider/Style/URI

Reply:
Neat